Gibe.DittoProcessors/Processors/AbsoluteUrlAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Umbraco.Core.Models;
+using Umbraco.Web;
+
+namespace Gibe.DittoProcessors.Processors
+{
+	public class AbsoluteUrlAttribute : TestableDittoProcessorAttribute
+	{
+
+		public override object ProcessValue()
+		{
+			var content = Value as IPublishedContent;
```

Comment:
I think we need to be clearer where we have one that uses a supplied [UmbracoProperty("x")] and one that uses the current node